operations/research-and-development/product/product-management-team-handbook.md

```diff
@@ -114,253 +114,253 @@ Before diving too deeply into docs, processes, and everything else - take this c
+* Market and Competitive Landscape
 * Team
   * People and Processes
 
 Activities:
 
-* Learn about our customers and users 
-  * Support 
-    * Customers: Zendesk \(Paul\) 
-      * Meet with Paul for Zendesk training and support overview 
-      * Join support standups for a week or two, and help out on support tickets 
+* Learn about our customers and users
+  * Support
+    * Customers: Zendesk \(Paul\)
+      * Meet with Paul for Zendesk training and support overview
+      * Join support standups for a week or two, and help out on support tickets
     * Community: GitHub, Forums \(Amy\)
-  * Customer success \(Meet with a CSM - Emily Hollinger, Stu Doherty, Rosie Lane\) 
+  * Customer success \(Meet with a CSM - Emily Hollinger, Stu Doherty, Rosie Lane\)
     * Overview of Customer Success team, who owns what
     * Current Customer Success processes
     * Typical challenges during customer onboarding
     * Join one CSM team meeting to meet the rest of the CS team
   * Customer calls
-    * Join calls with CSMs 
-    * Join calls with Sales team 
+    * Join calls with CSMs
+    * Join calls with Sales team
   * Feature requests
-    * Uservoice forums - [https://mattermost.uservoice.com/](https://mattermost.uservoice.com/)  
+    * Uservoice forums - [https://mattermost.uservoice.com/](https://mattermost.uservoice.com/)
     * [Customer request triage](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning.nodetail&quickFilter=97) 
-      * Track down estimates and early dev feasibility discussions on[ Customer Requests](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning&selectedIssue=CR-132)
+      * Track down estimates and early dev feasibility discussions on [Customer Requests](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning&selectedIssue=CR-132)
 * Learn about the market
-  * Sales \(Sales - Steve, Lance, Phillip & Solution Architects - Mark, John, Christian\) 
-    * Sales team overview 
-    * Sales team priorities 
-    * What competitors are we up against in sales 
-    * How does Sales team position the product 
-    * What are typical challenges in sales 
-    * Which sales do we win, which do we lose 
+  * Sales \(Sales - Steve, Lance, Phillip and Solution Architects - Mark, John, Christian\)
+    * Sales team overview
+    * Sales team priorities
+    * What competitors are we up against in sales
+    * How does Sales team position the product
+    * What are typical challenges in sales
+    * Which sales do we win, which do we lose
     * What are the most common questions on configurations and trials
-  * Marketing \(Lauren- Product Marketing Manager\)
-    * Marketing team overview 
-    * What are the marketing team’s priorities 
-    * What is our current messaging and positioning around Mattermost 
-    * What is needed from product managers
+  * Marketing \(Marketing team lead\)
```

Comment:
Confirm whether this is "Marketing team lead" or "Kendall - Marketing Operations Manager".